src/data/uint.rs

```diff
@@ -848,15 +834,10 @@ impl UInt {
 impl UInt {
     /// Returns the bit at the given bit position `pos`.
     /// 
-    /// This returns
```

Comment:
I think we have different mindsets when it comes to booleans and single bits. Since there are many people that do not implicitly define the conversions between bools and single bits I will go through all the single bit functions and make it explicit, and add on a commit.